app/Filament/Resources/ManagementInventoryResource.php

```diff
@@ -0,0 +1,154 @@
+use App\Models\ManagementInventory;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Support\Facades\Auth;
+
+class ManagementInventoryResource extends Resource
+{
+    protected static ?string $model = ManagementInventory::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-arrows-right-left';
+    protected static ?string $navigationGroup = 'Inventario';
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Forms\Components\Select::make('type')
+                    ->label('Tipo de Movimiento')
+                    ->options([
+                        'entrada' => 'Entrada',
+                        'salida' => 'Salida',
+                        'dañado' => 'Dañado',
+                        'devolución' => 'Devolución',
+                    ])
+                    ->required(),
+                Forms\Components\TextInput::make('quantity')
+                    ->label('Cantidad')
+                    ->numeric()
+                    ->required(),
+                Forms\Components\Textarea::make('description')
+                    ->label('Descripción')
+                    ->required()
+                    ->maxLength(65535),
+                Forms\Components\Hidden::make('model_type')
+                    ->default('App\Models\RawMaterialsInventory'),
+                Forms\Components\Select::make('model_id')
+                    ->label('Materia Prima')
+                    ->options(
+                        fn() => \App\Models\RawMaterialsInventory::query()
+                            ->orderBy('name')
+                            ->pluck('name', 'id')
+                            ->toArray()
+                    )
+                    ->searchable()
+                    ->required(),
+                Forms\Components\Hidden::make('created_by')
+                    ->default(fn() => Auth::user()->name),
+            ]);
+    }
+
+    public static function table(Table $table): Table
+    {
+        return $table
+            ->columns([
+                Tables\Columns\TextColumn::make('type')
+                    ->label('Tipo')
+                    ->badge()
+                    ->formatStateUsing(fn(string $state): string => match ($state) {
+                        'entrada' => 'Entrada',
+                        'salida' => 'Salida',
+                        'dañado' => 'Dañado',
+                        'devolución' => 'Devolución',
+                        default => $state,
+                    })
+                    ->color(fn(string $state): string => match ($state) {
+                        'entrada' => 'success',
+                        'salida' => 'warning',
+                        'dañado' => 'danger',
+                        'devolución' => 'info',
+                        default => 'gray',
```

Comment:
Esto lo podes agregar en el mismo enum de TypeInventoryManagementEnum tambien, toma el ejemplo de los depositos ahi yo aplique lo mismo.
```suggestion
                        'entrada' => 'success',
                        'salida' => 'warning',
                        'dañado' => 'danger',
                        'devolución' => 'info',
                        default => 'gray',
```